signer-service/src/constants/tokenConfig.js

```diff
@@ -20,6 +20,14 @@ const TOKEN_CONFIG = {
     decimals: 6,
     maximumSubsidyAmountRaw: '1000000', // 1 unit
   },
+  ars: {
+    tomlFileUrl: 'https://api.anclap.com/.well-known/stellar.toml',
+    assetCode: 'ARS\0',
+    assetIssuer: 'GCYE7C77EB5AWAA25R5XMWNI2EDOKTTFTTPZKM2SR5DI4B4WFD52DARS',
+    vaultAccountId: '6bE2vjpLRkRNoVDqDtzokxE34QdSJC2fz7c87R9yCVFFDNWs',
+    minWithdrawalAmount: '10000000000000',
+    maximumSubsidyAmountRaw: '100000000000000', // 100 unit ~ 0.1 USD @ Oct/2024
+  },
```

Comment:
What about the `pendulumCurrencyId`?

Reply:
Was added locally, not yet pushed at the time. Pushed now.